core/shared/src/main/scala/zio/prelude/Debug.scala

```diff
@@ -792,4 +795,8 @@ trait DebugSyntax {
   implicit class DebugOps[A](self: A) {
     def debug(implicit debug: Debug[A]): Debug.Repr = debug.debug(self)
   }
+
+  implicit final class DebugInterpolator(_sc: StringContext) {
+    def dbg(args: Debug.Repr*): String = _sc.s(args.map(_.toString): _*)
```

Comment:
Should we make this `d` so it feels more like a native interpolator?

Reply:
I also considered it... Plain Scala uses `s"..."` for `toString` interpolation and cats use `show"..."` for `show`. To me, `debug"..."` felt way too long and `d"..."` could be ambiguous or unclear to some people ("has this something to do with _decimal_, or what?").
I'm open to changing it, just unsure what would people like most.

Reply:
Did you want to do the `d` here?

Reply:
Oh, I thought you're fine with `dbg`. I can change it to `d` if you're sure it's better :+1:

Reply:
There is definitely subjectivity here. It seems like people preferred the `d` on Discord.

Reply:
I had made a survey, hadn't I :facepalm: 
:smile_cat: 
PR on the way!